src/System.Private.ServiceModel/tools/scripts/WCFCustomSetup.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+Echo Set Up test services
+REM
+
+REM ServiceUri is set, then the service is not local hosted
+if NOT "%ServiceUri%"=="" (
+  goto :done
+)
+
+REM If it's already running, we will not set up/clean up service
+tasklist /FI "imagename eq SelfHostedWCFService.exe" | findstr /i /C:"SelfHostedWCFService.exe" 1>nul
+if [%ERRORLEVEL%]==[0] (
+    goto :PingService
+)
+
+REM run the start wcf service elevated
+start /D %~dp0 RunElevated.vbs StartWCFService.cmd
+Echo WCFServiceCleanupNeeded
```

Comment:
Will be removed as we have decided to use exit code to decide if clean up is needed.